charts/astronomer/templates/houston/api/houston-deployment.yaml

```diff
@@ -58,6 +63,13 @@ spec:
                 secretKeyRef:
                   name: astronomer-bootstrap
                   key: connection
+            # we need to hardcode it to be able to upgrade from prisma1 to prisma2
+            - name: SCHEMA_NAME
+              value: "houston$default"
```

Comment:
What is this syntax? what is `$default`? Can `default` be renamed to be more specific? (EG: default_namespace, default_podname)

Reply:
That is a Postgres-ism. Its literally what you would use in a cli command for the database as well.

Reply:
@danielhoherd it's old prisma1 artifact... so prisma2 supports custom schema_name, we need this change to have ability to migrate existing platform to new platform with prisma2 in safe way.